src/Provider/Nominatim/Nominatim.php

```diff
@@ -123,7 +127,7 @@ public function reverseQuery(ReverseQuery $query): Collection
      *
      * @return Location
      */
-    private function xmlResultToArray(\DOMElement $resultNode, \DOMElement $addressNode): Location
+    private function xmlResultToArray(\DOMElement $resultNode, \DOMElement $addressNode, string $attribution = 'Data © OpenStreetMap contributors, ODbL 1.0. http://www.openstreetmap.org/copyright', bool $reverse): Location
```

Comment:
Will the default value ever have an effect here? 

See https://3v4l.org/F1LKl

Let's remove the default value.

Reply:
👍